### PR TITLE
String fixes. 

### LINKED ISSUE #725: Update copyright year in footer?
Randomly spotted on the site, not sure if it needs updating to **2019** or not...

> Portions of this content are Ⓒ **1998-2018** by individual mozilla.org contributors. 
Content available under a Creative Commons license.

https://github.com/mozilla/blurts-server/blob/ecc85281efd054343d7f9dc9e862805a36d367b1/locales/en/app.ftl#L243-L246

### LINKED ISSUE #741: Missing word in one of the new FxA strings
It looks like a word is missing in this string `fxa-what-to-do-blurb-2`

`Hackers may try to that same password and your email to get into other accounts. ` should probably be `Hackers may try to *reuse* that same password and your email to get into other accounts.`

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Punctuation updates requested by content. 
- Fixes #741 
- Fixes #725

### CODE REVIEW

File: views/partials/footer.hbs [hunk 1]
```diff
@@ -33,7 +33,10 @@
             </ul>
           </div>
           <div class="row legal">
-            <p class="half copyright-info">{{{fluentFormat req.supportedLocales "copyright-info"}}}</p>
```
Comment (pdehaan) [inline views/partials/footer.hbs]: Do we need to delete the "copyright-info" string from the .ftl file?

Comment (flodolo) [reply views/partials/footer.hbs]: Yes, we should

File: locales/en/app.ftl [hunk 2]
```diff
@@ -446,3 +446,8 @@ fxa-what-to-do-blurb-4 =
   on your phone and computer so you don’t have to remember them all.
 
 fb-landing-headline = Was your info exposed in the { $breachName } data breach?
+
+copyright = 
+  Portions of this content are { $years } by individual mozilla.org contributors.
```
Comment (flodolo) [inline locales/en/app.ftl]: Can we have only the final year exposed as a variable? Right now the code is making a lot of assumptions on spacing, separators, etc.

```
copyright = 
  Portions of this content are © 1998-{ $year } by individual mozilla.org contributors.
```

Comment (lesleyjanenorton) [reply locales/en/app.ftl]: Yep, will do!

**Pull request merged**